6-anthos-install/README.md

```diff
@@ -512,3 +512,23 @@ Example:
 
     kubectl apply -n transactions --context ${CTX_2} -f ${HOME}/terraform-example-foundation-app/6-anthos-install/db-scripts/populate-ledger-db.yaml
     ```
+
+# Troubleshooting
+
+1. If get the error that you are unable to access git repository:
+```
+fatal: unable to access 'https://source.developers.google.com/p/prj-bu1-c-app-cicd-XXXX/r/root-config-repo/': The requested URL returned error: 400
```

Comment:
@vfigueiredo-cit did this happen with `gcloud source repo clone` or regular `git clone`. gcloud commands usually handles that token refresh for us.

Reply:
@bharathkkb  happen with `gcloud source repo clone`. Because the git version used in this repository that we have to clone is 1.8.3.1 and gcloud can handle authentication with 2.0.1 or later.
When we try to clone the gcloud source repo, get the warning:
```
WARNING:          You are using a Google-hosted repository with a
         git version 1.8.3.1 which is older than 2.0.1. If you upgrade
         to 2.0.1 or later, gcloud can handle authentication to
         this repository. Otherwise, to authenticate, use your Google
         account and the password found by running the following command.
          $ gcloud auth print-access-token
```
This warning already gives the steps to solve the problem, but I think it would be a good idea to also put it in the troubleshooting.

Reply:
@vfigueiredo-cit and @bharathkkb 
git version issue should be solved, git 1.8 is the default in centos 7, while centos 8 has git 2.18 default.
When this PR was opened bastion-host module was using centos 7 as default, since then it has been [updated](https://github.com/terraform-google-modules/terraform-google-bastion-host/commit/090c0ea0254cbd31434fc974f6219e2c3c3b8d86#diff-05b5a57c136b6ff596500bcbfdcff145ef6cddea2a0e86d184d9daa9a65a288eL28-R28) to centos 8 on August 24

If this was the only issue, this PR can be closed, furthermore, I have commented on the linked issue #85 to add some more centext